### PR TITLE
Feat: Added Wishlist Page

### PR DESCRIPTION
**Pull request by AshwiniSetty**
This is the Wish List Page.

Preview: https://deploy-preview-5--minwii-stores.netlify.app/

### CODE REVIEW

File: pages/wishlist_page.html [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+    <meta name="viewport" content="width=device-width>, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/dd0d05bba7.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Minwii | Wishlist</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <nav>
```
Comment (Ayesha-7) [inline pages/wishlist_page.html:15]: good use of semantic element

**Review by Ayesha-7 — APPROVED** [review]
Looks good to merge.

File: pages/wishlist_page.html (hunk 1 above)
Comment (AshwiniSetty) [reply pages/wishlist_page.html:15]: Thank you

**Review by dev-enforced — APPROVED** [review]
Good use of the utility classes for setting up this wishlist part of Ecommerce. It is responsive too!!Good work @AshwiniSetty

**Review by khushijohri2001 — COMMENTED** [review]
Seems good to me. Everything looks fine👍

**Pull request merged**